recipe/meta.yaml

```diff
@@ -1,37 +1,35 @@
 {% set name = "healpix_cxx" %}
-{% set version = "3.31" %}
-{% set sha256 = "ddf437442b6d5ae7d75c9afaafc4ec43921f903c976e25db3c5ed5185a181542" %}
+{% set version = "3.70" %}
+{% set sha256 = "8841f171f1e22e75ea130e12e5cdc5bcf85dbec79f9f67dd1bf27e99fd20b6d1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: Healpix_{{ version }}_2016Aug26.tar.gz
-  url: https://downloads.sourceforge.net/project/healpix/Healpix_{{ version }}/Healpix_{{ version }}_2016Aug26.tar.gz
+  fn: Healpix_{{ version }}_2020Jul23.tar.gz
+  url: https://downloads.sourceforge.net/project/healpix/Healpix_{{ version }}/Healpix_{{ version }}_2020Jul23.tar.gz
   sha256: {{ sha256 }}
 
 build:
-  number: 1003
+  number: 0
   skip: true  # [win]
 
 requirements:
   build:
-    - autoconf
-    - {{ compiler('fortran') }}
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - make  # [not win]
   host:
     - cfitsio
+    - openmp
   run:
     - cfitsio
+    - openmp
 
 test:
   commands:
     - test -f ${PREFIX}/lib/libhealpix_cxx.a
-    - test -f ${PREFIX}/lib/libc_utils.a
-    - test -f ${PREFIX}/lib/libcxxsupport.a
-    - test -f ${PREFIX}/lib/libfftpack.a
```

Comment:
Did upstream deprecate these static libs?

Reply:
libc_utils and libcxxsupport are no longer built, and fftpack has been replaced by the compiled-in pocketfft in [version 3.50](https://healpix.sourceforge.io/html/install.htm#install:libsharp:config)